pkg/process/exec.go

```diff
@@ -5,10 +5,12 @@ package process // import "storj.io/storj/pkg/process"
 
 import (
 	"fmt"
+	"github.com/spf13/cobra"
```

Comment:
this needs to move outside of std lib or the linter will complain

Reply:
Already fixed 👍